lib/v2/screens/dashboard/components/tokens_cards/tokens_cards.dart

```diff
@@ -11,11 +11,17 @@ import 'package:seeds/v2/screens/dashboard/interactor/viewmodels/bloc.dart';
 
 import '../currency_info_card_widget.dart';
 
-class TokenCards extends StatelessWidget {
+class TokenCards extends StatefulWidget {
   const TokenCards({Key? key}) : super(key: key);
 
+  @override
+  _TokenCardsState createState() => _TokenCardsState();
+}
+
+class _TokenCardsState extends State<TokenCards> with AutomaticKeepAliveClientMixin {
```

Comment:
`AutomaticKeepAliveClientMixin` never seen this before, cool

Reply:
So great